packages/protocol/src/data/inscription.ts

```diff
@@ -25,14 +25,19 @@ export interface Inscription {
   inscriptionValidation: InscriptionValidation[];
   boolean: boolean;
   inscriptionActionArgs: InscriptionActionArgs;
+  void: Void;
   inscriptionSaveRequest: InscriptionSaveRequest;
   callableStart: CallableStart[];
   scriptingDataArgs: ScriptingDataArgs;
   variableInfo: VariableInfo;
   errorMeta: ErrorMeta[];
   eventCodeMeta: EventCodeMeta[];
   roleMeta: RoleMeta[];
-  connectorRef: ConnectorRef;
+  connectorRef: ConnectorRef[];
+  string: string[];
```

Comment:
string is very common. Better to use tables?

Reply:
and databases?

Reply:
This is the generated code from the inscription schema.
I think it calls the parameter like the type, and as I return a list of strings (e.g. as db names) it is called string here...
But this Inscription type here is anyways only here, so the types are correctly exported (or generated) I think. So it is not directly used.